README.md

```diff
@@ -59,3 +59,59 @@ Have you found any issues or missing features? Let us know via [GitHub issues](h
 ## License
 
 Iterative Provider is licensed under the [Apache 2.0 License](LICENSE).
+
+## Development
+
+### Install Go 1.17+
+
+Refer to the [official documentation](https://golang.org/doc/install) for specific instructions.
+
+### Clone the repository
+
+```console
+git clone https://github.com/iterative/terraform-provider-iterative
+cd terraform-provider-iterative
+```
+
+### Install the provider
+
+Build the provider and install the resulting binary to the [local mirror directory](https://www.terraform.io/docs/cli/config/config-file.html#implied-local-mirror-directories):
+
+```console
+make install
+```
+
+### Create a test file
+
+Create a file named `main.tf` on an empty directory with the follwing contents:
```

Comment:
Typo: follwing -> following
Create a file on a directory -> create a file in a directory

Reply:
```suggestion
Create a file named `main.tf` in an empty directory with the following contents:
```